lib/constants.js

```diff
@@ -317,6 +317,16 @@ export const LANGUAGES = [
     mime: 'text/x-mysql',
     short: 'mysql'
   },
+  {
+    name: 'N-Quads',
+    mode: 'n-quads',
+    mime: 'application/n-quads'
+  },
+  {
```

Comment:
Can we only add one of these as to try not and overload the language selector? 

I would prefer N-Triples, since according to the CodeMirror docs:
> The N-Triples mode also works well with on N-Quad documents.

Reply:
Sure, I understand your point.
I have made the changes.